blackbox/docs/sql/data_types.txt

```diff
@@ -123,6 +123,33 @@ its within the corresponding range. Example::
     ... );
     CREATE OK, 1 row affected (... sec)
 
+NaN, Infinity, -Infinity
+------------------------
+
+Since numeric types in CrateDB have corresponding Java types, ``float`` and
```

Comment:
i think the reason is not relevant. the point is that this is a standard.  see docs at https://www.postgresql.org/docs/current/static/datatype-numeric.html

maybe also mention the behaviour of NaN regarding computation